docs/book/user-guide/advanced-guide/use-failure-success-hooks.md

```diff
@@ -119,7 +119,7 @@ First, ensure that you have the OpenAI integration installed and have stored you
 
 ```shell
 zenml integration install openai
-zenml secret create openai --api_key=<YOUR_API_KEY>
+zenml secret create openai --values='{"api_key":"<YOUR_API_KEY>"}'
```

Comment:
in this case it isn't even about keeping the documentation about the current CLI option unchanged. You should just look at the two forms and ask yourself "which of the two would I rather use ?". I, for one, would definitely not like having to type the extra brackets and quotes needed for JSON when the previous option is so much cleaner and easier. Wouldn't you agree ?